### PR TITLE
Add a rich text Twig component

### PR DESCRIPTION
**Pull request by m-vo**
This introduces a `_rich_text` component as outlined in #6783.

This way you do not have to repeat yourself with things like this but can change it at one place:
```twig
<div class="rte">
    {{ text|csp_inline_styles|insert_tag|encode_email|raw }}
</div>
```

It should also make updates smoother when we add yet another filter. :stuck_out_tongue:

### CODE REVIEW

File: core-bundle/contao/templates/twig/component/_rich_text.html.twig [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{#
+    This component outputs rich text content, that was typically composed by
+    an editor in the backend using the tinyMCE editor.
+
+      <div class="rte">
+        <p>Lorem ipsum</p>
+      </div>
+
+    Mandatory variables:
+        @var string text
```
Comment (ausi) [inline core-bundle/contao/templates/twig/component/_rich_text.html.twig]: Should we add a note that `text` gets output `|raw` and therefore needs to be trusted?

File: core-bundle/contao/templates/twig/component/_rich_text.html.twig [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    Optional variables:
+        @var \Contao\CoreBundle\String\HtmlAttributes attributes
+
+    Note:
+        You can either group all variables in an object named "rich_text" or
+        all directly in the context. Grouping under an object has precedence.
+#}
+
+{% trans_default_domain "contao_default" %}
+
```
Comment (ausi) [inline core-bundle/contao/templates/twig/component/_rich_text.html.twig]:
```suggestion
```

This does not seem to be needed here?

Comment (m-vo) [reply core-bundle/contao/templates/twig/component/_rich_text.html.twig]: You're absolutely right. Copy and paste error. :slightly_smiling_face:

File: core-bundle/contao/templates/twig/component/_rich_text.html.twig (hunk 1 above)
Comment (m-vo) [reply core-bundle/contao/templates/twig/component/_rich_text.html.twig]: Good point, added in 4b3dc38.

**Review by ausi — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Pull request merged**